x-pack/legacy/plugins/infra/public/pages/logs/log_entry_categories/page_results_content.tsx

```diff
@@ -0,0 +1,248 @@
+
+  const { services } = useKibana<{}>();
+
+  const showLoadDataErrorNotification = useCallback(
+    (error: Error) => {
+      // eslint-disable-next-line no-unused-expressions
+      services.notifications?.toasts.addError(error, {
+        title: loadDataErrorTitle,
+      });
+    },
+    [services.notifications]
+  );
+
+  const {
+    getLogEntryCategoryDatasets,
+    getTopLogEntryCategories,
+    isLoadingLogEntryCategoryDatasets,
+    isLoadingTopLogEntryCategories,
+    logEntryCategoryDatasets,
+    topLogEntryCategories,
+  } = useLogEntryCategoriesResults({
+    categoriesCount: 25,
+    endTime: categoryQueryTimeRange.timeRange.endTime,
+    filteredDatasets: categoryQueryDatasets,
+    onGetTopLogEntryCategoriesError: showLoadDataErrorNotification,
+    sourceId,
+    startTime: categoryQueryTimeRange.timeRange.startTime,
+  });
+
+  const handleQueryTimeRangeChange = useCallback(
+    ({ start: startTime, end: endTime }: { start: string; end: string }) => {
+      setCategoryQueryTimeRange(previousQueryParameters => ({
+        ...previousQueryParameters,
+        timeRange: stringToNumericTimeRange({ startTime, endTime }),
+        lastChangedTime: Date.now(),
+      }));
+    },
+    [setCategoryQueryTimeRange]
+  );
+
+  const handleSelectedTimeRangeChange = useCallback(
+    (selectedTime: { start: string; end: string; isInvalid: boolean }) => {
+      if (selectedTime.isInvalid) {
+        return;
+      }
+      setSelectedTimeRange({
+        startTime: selectedTime.start,
+        endTime: selectedTime.end,
+      });
+      handleQueryTimeRangeChange(selectedTime);
+    },
+    [setSelectedTimeRange, handleQueryTimeRangeChange]
+  );
+
+  const handleAutoRefreshChange = useCallback(
+    ({ isPaused, refreshInterval: interval }: { isPaused: boolean; refreshInterval: number }) => {
+      setAutoRefresh({
+        isPaused,
+        interval,
+      });
+    },
+    [setAutoRefresh]
+  );
+
+  const isFirstUse = useMemo(() => setupStatus === 'hiddenAfterSuccess', [setupStatus]);
+
+  const hasResults = useMemo(() => topLogEntryCategories.length > 0, [
+    topLogEntryCategories.length,
+  ]);
+
+  useEffect(() => {
+    getTopLogEntryCategories();
+  }, [getTopLogEntryCategories, categoryQueryDatasets, categoryQueryTimeRange.lastChangedTime]);
+
+  useEffect(() => {
+    getLogEntryCategoryDatasets();
+  }, [getLogEntryCategoryDatasets, categoryQueryTimeRange.lastChangedTime]);
+
+  useInterval(() => {
+    fetchJobStatus();
+  }, JOB_STATUS_POLLING_INTERVAL);
+
+  useInterval(
+    () => {
+      handleQueryTimeRangeChange({
+        start: selectedTimeRange.startTime,
+        end: selectedTimeRange.endTime,
+      });
+    },
+    autoRefresh.isPaused ? null : autoRefresh.interval
+  );
+
+  return (
+    <ResultsContentPage>
+      <EuiPageBody>
+        <EuiFlexGroup direction="column">
+          <EuiFlexItem grow={false}>
+            <EuiPanel paddingSize="l">
```

Comment:
```suggestion
            <EuiPanel paddingSize="m">
```
But I saw that the log rate panels also have large paddings - maybe we should change them together?

Reply:
Yes, I'll change both in a separate PR if you don't mind.